draft-ietf-teep-protocol.md

```diff
@@ -1156,29 +1161,57 @@ or Error message is generated only after completing the Update Procedure.
 
 # Ciphersuites {#ciphersuite}
 
-A ciphersuite consists of an AEAD algorithm, a MAC algorithm, and a signature
-algorithm.
-Each ciphersuite is identified with an integer value, which corresponds to
-an IANA registered
-ciphersuite (see {{ciphersuite-registry}}. This document specifies two ciphersuites.
-
-| Value | Ciphersuite                                    |
-|     1 | AES-CCM-16-64-128, HMAC 256/256, X25519, EdDSA |
-|     2 | AES-CCM-16-64-128, HMAC 256/256, P-256, ES256  |
-
-A TAM MUST support both ciphersuites.  A TEEP Agent MUST support at least
-one of the two but can choose which one.  For example, a TEEP Agent might
-choose ciphersuite 2 if it has hardware support for it.
-
-Any ciphersuites without confidentiality protection can only be added if the
-associated specification includes a discussion of security considerations and
-applicability, since manifests may carry sensitive information. For example,
-Section 6 of {{I-D.ietf-teep-architecture}} permits implementations that terminate
-transport security inside the TEE and if the transport security provides
-confidentiality then additional encryption might not be needed in the manifest
-for some use cases. For most use cases, however, manifest confidentiality will
-be needed to protect sensitive fields from the TAM as discussed in Section 9.8
-of {{I-D.ietf-teep-architecture}}.
+TEEP protocol uses COSE as protection mechanisms of TEEP messages.
+After QueryResponse, the selected cryptographic algorithm is used in the TEEP messages: Install, Success and Error.
+To negotiate the chosing cryptographic mechanisms and algorithms, TEEP protocol defines the ciphersuite structure.
+
+~~~~
+suite = [
```

Comment:
Can we call this "ciphersuite" instead of just "suite"?

Reply:
Yes, we can call "ciphersuite".